lite/tests/detection/conftest.py

```diff
@@ -309,9 +403,9 @@ def false_negatives_single_datum_detections() -> list[Detection]:
 
 
 @pytest.fixture
-def false_negatives_two_datums_one_empty_low_confidence_of_fp_detections() -> list[
-    Detection
-]:
+def false_negatives_two_datums_one_empty_low_confidence_of_fp_detections() -> (
+    list[Detection]
+):
```

Comment:
Why make this change? It seems like this is done across this file.

Reply:
this fixture returns a list of Detection, not just one Detection. it was throwing a type error in my linter

Reply:
It looks like it and the others were already defined as lists. Why the change from `-> list[Detection]` to `-> (list[Detection])`